src/cogs/core_commands.py

```diff
@@ -175,14 +175,14 @@ async def autoclose(self, ctx, option: str = "off", channel: discord.TextChannel
     @commands.command(name="auto_message", aliases=["am"])
     @commands.has_role(config.roles['admin'])
     async def auto_message(self, ctx, channel: discord.TextChannel):
-        """Sends a message asking if the ticket can be closed. Does not contribute to AC checks"""
+        """Asks if ticket can be closed. Does not contribute to AC checks"""
         try:
             user_id = db.get_user_id(channel.id)
         except ValueError as e:
             return await ctx.channel.send(e.args[0])
 
         member = ctx.guild.get_member(int(user_id))
-        message = f"If that is all we can help you with {member.mention}, please close this ticket."
+        message = f"If that is all we can help you with {member.mention}, please close this ticket.\n||I am a bot and this action was performed automatically||"
```

Comment:
Not sure if this is needed, it may be useful though

Reply:
How about no "I am a bot" and no spoiler tags? so just " (this action was performed automatically)".

Reply:
sure